src/otp-input/otp-input.tsx

```diff
@@ -125,13 +157,8 @@ export const OtpInput = ({
         if (!event.clipboardData) return;
 
         const pastedValue = event.clipboardData.getData("text");
-        const pastedValueArr = pastedValue.split("");
-        if (pastedValue && validateUserInput(pastedValue, numOfInput)) {
-            setOtpValues(pastedValueArr);
-            if (onChangeRef.current) {
-                onChangeRef.current(pastedValueArr);
-            }
-        } else {
+
+        if (!handleInputFullValue(pastedValue)) {
```

Comment:
can handlePaste also include the strip logic? my thought process is even on desktop, user can also copy the whole OTP including the prefix

Reply:
ok just need to clarify
- autofill otp expects field to be empty, ie if i already key in 1 number, the autofill won't work
- paste current behaviour is to completely overwrite if the clipboard value is valid
- do we want to follow either one of the behaviours, or to have them maintain their own behaviour?